oaknational/Oak-Web-Application
# [CUR-237] Curriculum visualiser mobile optimisation

## Description

Music year: {owa_music_year}

- List of changes
Designed according to Figma here for mobile: https://www.figma.com/file/3cz9fb0ObFDLv1c212OtJv/Final-designs---Curriculum-squad?node-id=811%3A357218&mode=dev

To do:
[] Look into fixing app header
[] Styling / spacing finishing touches
[] Adding offset to scrolling to year group to account for the fixed header

Desktop should remain the same

## How to test

1. Go to {owa_deployment_url}
2. Click on any subject phase combination in mobile
3. You should see the visual changes according to Figma

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
